_rules/SC2-4-9-links-with-identical-accessible-names-have-same-purpose.md

```diff
@@ -0,0 +1,234 @@
+---
+name: Links with identical accessible names have same purpose
+test_type: atomic
+
+description: |
+  This rule checks that identical accessible names are only used for links that have the same purpose
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [exposed to assistive technologies](#exposed-to-assistive-technologies) and that have the same [accessible name](#accessible-name).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to resources that fulfil the same purpose in relation to the purpose indicated by the [accessible names](#accessible-name) of the link.
+
+**Note:** Resolving the links includes potential redirects.
+
+**Note:** Web pages and documents (e.g. PDFs, office formats etc.) may fulfil the same purpose in relation to the link, even if the resources:
+* are located on different URLs, including different domains
+* present different navigation options, e.g. through bread crumbs or local sub menus
+* contain different amounts of information and/or differently worded information
+* are using different layouts.
+
+**Note:** If the same content is presented in different formats, the format itself is often part of the link purpose, e.g. an article as both HTML and PDF.
+
+**Note:** If the [normalized](#url-normalisation) value of the ´href´ is identical, the resources are identical, thus fulfilling the same purpose. [Relative URLs](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#relative-urls) first need to be resolved to full URLs before doing the comparison.
+
+## Assumptions
+
+* This rule assumes that the purpose of the links for links with identical link texts would not be ambiguous to users in general, which is the exception mentioned in [Understanding Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html).
```

Comment:
There's also an assumption that the accessible name is the same as the link text. I'm not sure if that should be an assumption, or if the rule should be changed to compare the visible text, instead of the accessible name.